packages/plugins/model-define/detectron-maskrcnn-model-define/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@pipcook/plugins-detectron-maskrcnn-model-define",
+  "version": "0.5.10",
+  "description": "",
+  "main": "dist/index",
+  "types": "dist/index",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "npm run clean && npm run compile",
+    "clean": "rm -rf ./dist && rm -rf tsconfig.tsbuildinfo",
+    "compile": "tsc -b tsconfig.json && cp -r ./src/config ./dist/"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@pipcook/boa": "^0.5.10",
+    "@pipcook/pipcook-core": "^0.5.16"
+  },
+  "gitHead": "1e21ad0799e221dcf4b7aa3a3b2187d5b8d8b865",
+  "devDependencies": {
+    "@types/jasmine": "^3.5.7",
+    "nyc": "14.1.1"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "pipcook": {
+    "category": "modelDefine",
+    "datatype": "image",
+    "method": "segmentation"
```

Comment:
Do we decide to use the "method" now?

Reply:
Or I can remove method for now. Anyway, this attribute is used for MLAPP

Reply:
Yea, I'm +1 on removing this here, and we will have updates by MLAPP/PipApp.

Reply:
Have removed them :)